### ISSUE TITLE
What do you call the custom card in Lovelace?

### ISSUE THREAD

**Issue by broyuken** [Samantha-uk/zigzag-v1#2]
What do you name it to add the custom card in Lovelace? I’ve tried
 
custom:zigzag 
custom:zigzag-card
custom:zigzag-panel

but they don’t work.

**Comment by boralyl** [comment]
I believe you want to use `custom:custom-card-zigzag`

**Comment by broyuken** [comment]
Nope, that didn't work

**Comment by boralyl** [comment]
Not sure which method you are using for lovelace (ui or yaml) but I just tested it out and `custom:custom-card-zigzag` is correct.  Although I think this works much better as a panel than a card, especially if you have more than a few devices.  If you wanted to use the card and use yaml mode all I had to do was:

```yaml
cards:
  - type: custom:custom-card-zigzag
```

Don't forget to add the javascript module either in the UI or in configuration.yaml:

```yaml
lovelace:
  mode: yaml
  resources:
    - url: /local/zigzag/zigzag-card.js
      type: module
```

**Comment by broyuken** [comment]
I ended up moving the js file up a directory, it was in /config/www/community with all my others, and even though I referenced it explicity in resources it didn't matter until I changed it to /local/zigzag/zigzag-card.js. Anyways, now it works. Thanks for clarifying. The documentation should probably be updated to include this. 

Also, side note I am using this as a panel, I just didn't want it on the side. It's a new view in panel mode.
